### PR TITLE
Try conda environment

### PR DESCRIPTION
**Pull request by lilyminium**
Try different things relating to #82.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -18,11 +18,12 @@ env:
   - SPHINX_DIR=doc/
   - HTML_DIR=doc/build/html
   - PYTHON_VERSION=3.6
```
Comment (kain88-de) [inline .travis.yml:20]: try python version 3.8

**Comment by lilyminium** [comment]
Thanks @kain88-de, I see you’ve done that :-) the previous builds were actually failing due to astropy’s ci-helpers hanging after building from the environment.yml, wasn’t sure if that was a bug on their, ours, or the Travis end. Any ideas?

**Comment by kain88-de** [comment]
Conda tripping over itself. Hard to say why.

**Comment by kain88-de** [comment]
I have no idea why there is a segfault. It is also really hard to see where the code has problems because sphinx is not very verbose here. No idea what is happening. But I do not like that our conda packages segfault.

**Comment by lilyminium** [comment]
Closing due to now having to work out how to do this via GH-actions.

**Pull request closed without merging**